documentation/docs/05-misc/02-accessibility-warnings.md

```diff
@@ -52,16 +52,18 @@ Enforce that `autofocus` is not used on elements. Autofocusing elements can caus
 
 ## `a11y-click-events-have-key-events`
 
-Enforce `on:click` is accompanied by at least one of the following: `on:keyup`, `on:keydown`, `on:keypress`. Coding for the keyboard is important for users with physical disabilities who cannot use a mouse, AT compatibility, and screenreader users.
+Enforce that visible, non-interactive elements with an `on:click` event are accompanied by a keyboard event handler.
 
-This does not apply for interactive or hidden elements.
+Users should first consider whether an interactive element might be more appropriate such as a `<button type="button">` element for actions or `<a>` element for navigations. These elements are more semantically meaningful and will have built-in key handling. E.g. `Space` and `Enter` will trigger a `<button>` element's navigation while `Space` will focus an `<a>` element and `Enter` will trigger an `<a>` element's navigation.
```

Comment:
The note about `Space` regarding links seems wrong. It should just scroll the page.

Also the wording around button using "navigation" sounds off, buttons can do all sorts of things, so I would not further qualify that. Likewise the behavior of links can be changed.

Would recommend simplifying the last sentence to something like:
```md
E.g. `Space` and `Enter` will trigger a `<button>`  and `Enter` will trigger an `<a>` element.
```